lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -96,6 +96,7 @@ uint32_t findLayerIndex(const UsdPrim& prim, const SdfLayerHandle& layer)
         // to find the layer
         for (SdfLayerRefPtr const& l : layerStack->GetLayers()) {
             if (l == layer) {
```

Comment:
I would prefer an explicit set to false at line 105 when not found, for clarity.

Reply:
This would also avoid the need for the explicit reset of the variable to false in the caller, when it tries to call this a 2nd time, below.